spec/models/like_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject { Like.new }
+  before { subject.save }
+
+  it 'like_counter method should raise an error without post' do
+    expect { subject.like_counter }.to raise_error(NoMethodError)
+  end
+end
```

Comment:
- [ ] Let us endeavor to be thorough in our specs. Kindly add positive scenarios where the likes save works and the like counter of the related post is updated.